docs/FLYSKY_FRM303.md

```diff
@@ -2,7 +2,7 @@
 
 ([back to main page](../README.md))
 
-The FlySky FRM303 transmitter module is a nearly perfect option for building mLRS equipment in the 2.4 GHz frequency range. Its main drawback is its price. It also has a small MCU which limits the available mLRS features. However, no hardware tweaking is required (although mods like the OLED mod are possible).
+The FlySky FRM303 transmitter module is a nearly perfect option for building mLRS equipment in the 2.4 GHz frequency range. Its main drawback is its price. It also has a small MCU which limits the available mLRS features. However, no hardware tweaking is required (although mods, like the OLED mod and mLRS-WiFi-bridge, are possible. See bottom of the page for more about optional modifications).
```

Comment:
Small note - could we move these last line out of parentheses.  "no hardware tweaking is required.  Although mods..."

Reply:
Done.